hoomd/pytest/test_snapshot.py

```diff
@@ -125,6 +125,128 @@ def test_configuration(s):
         assert s.configuration.dimensions == 3
 
 
+def test_wrap(s):
+
+    def generate_outside(box, interior_points, multipliers, initial_images):
+        """Generate test cases from interior points by adding box vectors."""
+        # construct unit cell from box vectors
+        # see
+        # hoomd-blue.readthedocs.io/en/latest/...
+        # ...package-hoomd.html#hoomd.Box.from_matrix
```

Comment:
Don't split URLs like this, it makes them too difficult to copy-paste. If the line is too long for flake8's standards, you can add a "noqa" flag to this comment. See here for an example: https://stackoverflow.com/a/47876831